modin/core/storage_formats/pandas/query_compiler.py

```diff
@@ -3472,31 +3472,52 @@ def _groupby_internal_columns(self, by, drop):
 
         Returns
         -------
-        by : list of BaseQueryCompiler, column or index label, or Grouper
+        external_by : list of BaseQueryCompiler and arrays
```

Comment:
The name of the function no longer corresponds to what it does (not just internal columns).

Reply:
renamed to `_groupby_separate_by`